storybook/cypress/e2e/map-controls.cy.ts

```diff
@@ -56,6 +59,7 @@ describe("Map Controls", () => {
   it('panel should have Dutch header "Kaartlagen" and close button "Verberg paneel Kaartlagen"', () => {
     cy.get("@toggleVisibilityButton")
       .click()
+      .wait(transitionDuration)
```

Comment:
Deze change is om de volgende onterecht falende snapshots te fixen:

<img width="1200" height="600" alt="Image" src="https://github.com/user-attachments/assets/dc4812cd-74a1-4cef-93cf-7fc56f4dae93" />